tests/tests.js

```diff
@@ -157,6 +157,19 @@ exports.defineManualTests = function (contentEl, createActionButton) {
         document.getElementById('compass_status').innerHTML = status;
     }
 
+    // Success callback for both watchHeading and getCurrentHeading
+    function success(a) {
+        var magneticheading = document.getElementById('magneticHeading');
```

Comment:
Thanks for catching it. I wonder why it appeared to be working when I was testing it...